src/main/java/com/google/sps/servlets/AvailabilityServlet.java

```diff
@@ -31,7 +56,27 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        response.setContentType("plain/text");
-        response.getWriter().println("To be implemented");
+        // Get the id of the tutor whose availability will be displayed.
+        String tutorID = Optional.ofNullable(request.getParameter("tutorID")).orElse(null);
```

Comment:
I don't really think it makes sense to use an Optional here if the default is null.

Reply:
That makes a lot of sense. I just removed it.